### PR TITLE
font test

### PR DESCRIPTION
**Pull request by maelle**
I'd like to not use Google Fonts for Fira Mono. But first let's decide on whether this looks good. :sweat_smile:

### CODE REVIEW

File: inst/pkgdown/templates/head.html [hunk 1]
```diff
@@ -12,6 +12,8 @@
 <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
 <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="/apple-touch-icon.png">
 
+<link rel="stylesheet" href="fonts.css" />
```
Comment (maelle) [inline inst/pkgdown/templates/head.html]: Eventually this css could live at ropensci dot org

**Comment by jeroen** [comment]
Currently this has the serif flavor, whereas we use the sans-serif flavor on ropensci.org.

**Comment by jeroen** [comment]
It still seems to have serif fonts for all the code blocks and bullet points? Maybe we should just stick with the default fonts if this is going to complicate things...

**Comment by maelle** [comment]
The errors I get are related to cross origin requests, I wonder whether the file fonts need to live somewhere at docs.ropensci.org for the whole thing to work.

**Comment by maelle** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#the_http_response_headers
https://support.cloudflare.com/hc/en-us/articles/200308847-Using-cross-origin-resource-sharing-CORS-with-Cloudflare
https://answers.netlify.com/t/access-control-allow-origin-policy/1813

**Pull request merged**